### DISCUSSION TITLE
[FR] Editing on entry should load draft for user if one exists

### DISCUSSION THREAD

**Discussion by leevigraham** [craftcms/cms#8839] [Ideas] [answered]
### Description

When a user edits an entry after they have create a draft the draft should load, not the original entry.

### Steps to reproduce

1. Create a new draft
2. Visit the entry index
3. Edit the entry
4. The original entry loads not the draft

![2020-06-03 17 48 36](https://user-images.githubusercontent.com/25124/83610336-a0e7b680-a5c2-11ea-8f42-fdfb5eee9add.gif)

### Additional info

- Craft version: 3.5 beta2

**Comment by mmikkel** [comment]
Ideally, I think this should only happen for drafts that were manually created. The auto-drafts are still a bit confusing/transparent and I think some users would definitely be even more confused if they opened up an entry and Craft served up their newest draft that was created automatically (perhaps days ago) and that they might not even be aware of, instead of the current entry version.

On a general note, making the visual difference between drafts and current versions more apparent could probably help with this.

**Comment by leevigraham** [comment]
@brandonkelly Not sure but this could be closed now given the 3.7 publishing updates?

**Reply by brandonkelly** [reply]
Yep!

**Comment by brandonkelly** [comment, marked as answer]
This is resolved in Craft 3.7 thanks to the publishing workflow improvements, where any unsaved changes to an entry are stored in a private “provisional draft”, which will be loaded automatically when the author returns to the entry later. (#7899)
